index.bs

```diff
@@ -7077,25 +7077,18 @@ values. Those values persist accros invocation of this algorithm.
 	<var>reduction gain</var>, for each sample of input, for a render
 	quantum of audio.
 
-	1. Let <var>threshold</var>, <var>knee</var> have the value of the
-		{{AudioParam}} of the same name, <a href="#db-to-linear">converted to linear unit</a> sampled at the time of
-		processing of this block (as <a>k-rate</a> parameters).
 
-	2. Let <var>ratio</var> have the value of the <code>ratio</code>
-		{{AudioParam}}, sampled at the time of processing of this block
-		(as a <a>k-rate</a> parameter).
-
-	3. Let <var>attack</var> and <var>release</var> have the value of
+	1. Let <var>attack</var> and <var>release</var> have the value of
 		the {{AudioParam}} of the same name, sampled at the time of
 		processing (those are <a>k-rate</a> parameters), mutiplied by the
 		sample-rate of the {{BaseAudioContext}} this
 		{{DynamicsCompressorNode}} is <a href="#associated">associated</a> with.
 
-	4. Let <var>detector average</var> be the value of the slot {{[[detector average]]}}.
+	1. Let <var>detector average</var> be the value of the slot {{[[detector average]]}}.
```

Comment:
I didn't change the names.  If I do, I'll have to change it in a few other places which I was kind of avoiding to minimize the number of changes and because those were the names in the original version.